### PR TITLE
Add pootle_fs resources class

### PR DESCRIPTION
**Pull request by phlax**
Adds a resource class for encapsulating fs plugins relationship to Stores/StoreFS

### CODE REVIEW

File: tests/pootle_fs/resources.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    assert str(resources) == "<FSResources(Project 0)>"
+
+
+@pytest.mark.django_db
+def test_project_resources_stores():
+    project = Project.objects.get(code="project0")
+    stores = Store.objects.filter(
+        translation_project__project=project)
+    assert list(FSResources(project).stores) == list(stores)
+    # mark some Stores obsolete - should still show
```
Comment (unho) [inline tests/pootle_fs/resources.py]: Is this comment about something not yet coded?

**Comment by unho** [comment]
lgtm aside that comment that is not clear to me.

File: tests/pootle_fs/resources.py (hunk 1 above)
Comment (phlax) [inline tests/pootle_fs/resources.py]: yep, a todo, ill add

File: tests/pootle_fs/resources.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    obsoleted = synced.filter(store__obsolete=True).first()
+    obsoleted.last_sync_hash = "FOO"
+    obsoleted.last_sync_revision = 23
+    obsoleted.save()
+    active = synced.exclude(store__obsolete=True).first()
+    active.last_sync_hash = "FOO"
+    active.last_sync_revision = 23
+    active.save()
+    synced = synced.exclude(last_sync_revision__isnull=True)
+    synced = synced.exclude(last_sync_hash__isnull=True)
```
Comment (ta2-1) [inline tests/pootle_fs/resources.py]: this is a test of course but i thought there are duplicated strings here..

File: pootle/apps/pootle_fs/resources.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+
+from pootle_fs.models import StoreFS
+from pootle_store.models import Store
+
+
+class FSResources(object):
```
Comment (ta2-1) [inline pootle/apps/pootle_fs/resources.py]: FSProjectResources

File: pootle/apps/pootle_fs/resources.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                        .exclude(last_sync_hash__isnull=True))
+
+    @property
+    def unsynced(self):
+        return (
+            self.tracked.filter(last_sync_revision__isnull=True)
+                        .filter(last_sync_hash__isnull=True))
+
+    @property
+    def trackable(self):
```
Comment (ta2-1) [inline pootle/apps/pootle_fs/resources.py]: trackable_stores

**Comment by ta2-1** [comment]
there are some nits. 
lgtm after they are addressed.

File: tests/pootle_fs/resources.py (hunk 2 above)
Comment (phlax) [inline tests/pootle_fs/resources.py]: which strings are duplicated?

Comment (ta2-1) [inline tests/pootle_fs/resources.py]: there are no duplicated strings but I could think so for a second.

```
synced = synced.exclude(
    last_sync_revision__isnull=True,
    last_sync_hash__isnull=True)
```

Comment (phlax) [inline tests/pootle_fs/resources.py]: i did it this way for a reason, that being that the ORM can do different things with in these 2 situations - altho not always clear to me exactly why.

basically i want an OR exclude, and im not 100% i would get that with the way you show

Comment (ta2-1) [inline tests/pootle_fs/resources.py]:
```
from django.db.models import Q
synced = synced.exclude(
    Q(last_sync_revision__isnull=True)
    | Q(last_sync_hash__isnull=True)
)
```
I always do `OR` via `Q()`, but I don't know if it works here?

Comment (phlax) [inline tests/pootle_fs/resources.py]: yep also `Q(~some_file=...)`

tbh i went with what i know works and is relatively simple

Comment (ta2-1) [inline tests/pootle_fs/resources.py]: I wonder why 
```
qs = qs.exclude(cond1)
qs = qs.exclude(cond2)
``` 
works as `OR`. It has to work as `AND`, doesn't it?

**Pull request merged**